plugins/module_utils/job.py

```diff
@@ -200,6 +205,9 @@ def _get_job_status(job_id="*", owner="*", job_name="*", dd_name=None, duration=
     # jls output: owner=job[0], name=job[1], id=job[2], status=job[3], rc=job[4]
     # e.g.: OMVSADM  HELLO    JOB00126 JCLERR   ?
     # listing(job_id, owner) in 1.2.0 has owner param, 1.1 does not
+    # jls output has expanded in zos 1.2.3 and later: jls -l -v shows headers
```

Comment:
Sorry for nitpicking, this is just a typo: `zos` instead of `zoau`.

Reply:
don't be sorry.... details like that look good when the customers don't see them.  I've fixed it.